dd-sdk-android-core/src/main/kotlin/com/datadog/android/core/DatadogCore.kt

```diff
@@ -227,6 +236,38 @@ internal class DatadogCore(
         }
     }
 
+    override fun setContextUpdateListener(featureName: String, listener: FeatureContextUpdateListener) {
```

Comment:
wanted to keep the same approach as for receiver but it makes more sense to move this logic into the `SdkFeature` itself.